raspi-config

```diff
@@ -1312,7 +1312,7 @@ do_fan() {
     whiptail --yesno "Would you like to enable fan temperature control?" $DEFAULT 20 60 2
     RET=$?
   else
-    RET=$1
+    RET=$(expr 1 - $1)
```

Comment:
There appears to me to be an issue with this code:

if you do `raspi-config nonint do_fan 1 14 60`, then `$1` is `1`. However, line 1317 expects `$RET` to equal `0` if the user is attempting to enable the fan.

The [documentation](https://www.raspberrypi.com/documentation/computers/configuration.html#fan-2) would indicate that we should expect the fan to be enabled if we provide `1`.

My change inverts the value of the first argument. I'm not sure that's the best way to handle this, but it would seem to be the least disruptive. It works for me when I make the change locally, whereas I have to do `raspi-config nonint do_fan 0 14 60` to enable the fan control with the current code.

Reply:
> The [documentation](https://www.raspberrypi.com/documentation/computers/configuration.html#fan-2) would indicate that we should expect the fan to be enabled if we provide `1`.

The documentation you've linked to says "0: enable fan" ?

Reply:
Could've sworn it said the opposite four months ago, but maybe I'm cracking. Thanks for the comment!